apps/judicial-system/web/src/routes/Prosecutor/DetentionRequests/DetentionRequests.tsx

```diff
@@ -28,6 +28,7 @@ export const DetentionRequests: React.FC = () => {
   const [cases, setCases] = useState<DetentionRequest[]>(null)
   const [isLoading, setIsLoading] = useState<boolean>(true)
   const uContext = useContext(userContext)
+  const isJudge = uContext.user.role === UserRole.JUDGE
```

Comment:
I wonder if it would make sense to either:

1. Move this page under a common route (i.e. not under Prosecutor), or
2. Have a separate page for judges under Judge routes.